doc/user/content/sql/create-index.md

```diff
@@ -145,22 +145,23 @@ CREATE MATERIALIZED VIEW active_customers AS
 
 CREATE INDEX active_customers_idx ON active_customers (guid);
 
+-- The above index speeds up the following filter:
 SELECT * FROM active_customers WHERE guid = 'd868a5bf-2430-461d-a665-40418b1125e7';
 
--- Using expressions
+-- An index whose key is an expression:
```

Comment:
We use the term _indexed expressions_ [here](https://materialize.com/docs/sql/create-index/#indexed-expressions-vs-stored-columns).